addons/web_editor/static/src/xml/snippets.xml

```diff
@@ -19,6 +19,10 @@
     </t>
     <t t-name="web_editor.customize_block_options_section">
         <we-customizeblock-options>
+            <we-alert t-if="!is_up_to_date">
+                <we-title>This snippet is outdated</we-title>
+                <p>If you wish to make changes we recommend you to drop the new version of this block and rebuild it with the new options</p>
```

Comment:
The translation of this sentence would be: "Si vous souhaitez apporter des modifications, nous vous recommandons de laisser tomber la nouvelle version de ce bloc et de le reconstruire avec les nouvelles options", which is the opposite of what you want to say :p